constructor/main.py

```diff
@@ -197,6 +197,15 @@ def main_build(dir_path, output_dir='.', platform=cc_platform,
             )
         )
 
+    # Add --no-rc option to CONDA_EXE command so that existing
+    # .condarc files do not pollute the installation process.
+    if exe_name == "conda-standalone" and Version(exe_version) >= Version("24.9.0"):
```

Comment:
```suggestion
    if exe_name == "conda-standalone" and exe_version and Version(exe_version) >= Version("24.9.0"):
```

Protect against `exe_version = None`.